packages/Popover/src/Popover.spec.tsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { render, cleanup } from '@testing-library/react';
-import Popover, { Align, Justify } from './Popover';
+import Popover from './Popover';
+import Align from './Align';
+import Justify from './Justify';
```

Comment:
Why not one unified `types` file? Feels like a lot to have a separate file for every "enum-like" type we have.

Reply:
@hswolff do you have a preference here?

Reply:
Not really. They're not exactly types tho, cuz they're also objects. But no preference.

Reply:
Another idea would be `props.ts` – export the props and the specific prop types like these and such.

I know we extend Popover's props elsewhere too, so there's that.

Reply:
I suppose my point is that it would be nice to have a uniform file across packages that exports these, rather than `Align.ts`, `Justify.ts`, `Variant.ts`, etc.

Reply:
Yeah, I agree with that sentiment. Not sure of what that is yet. We can think on it tho, but not block this PR with that answer.

Reply:
Fair enough – onward march!